### PR TITLE
Transfer dub to org

### PR DESCRIPTION
**Pull request by peduarte**

### CODE REVIEW

File: extensions/dub-link-shortener/CHANGELOG.md
Comment (jfkisafk) [inline extensions/dub-link-shortener/CHANGELOG.md:1]: Shouldn't we rename the directory from _extensions/dub-link-shortener/_ to _extensions/dub/_, corresponding to the extension name change?

**Review by jfkisafk — APPROVED** [review]

File: extensions/dub-link-shortener/CHANGELOG.md
Comment (peduarte) [reply extensions/dub-link-shortener/CHANGELOG.md:1]: yeah, I literally just did that haha

Comment (jfkisafk) [reply extensions/dub-link-shortener/CHANGELOG.md:1]: I see you did it in the following commit 🔥 ty ty!

File: extensions/dub/metadata/dub-link-shortener-1.png
Comment (jfkisafk) [inline extensions/dub/metadata/dub-link-shortener-1.png:1]: I believe the screenshots should be renamed from _dub-link-shortener-1.png_ to _dub-1.png_ and so on.

Comment (peduarte) [reply extensions/dub/metadata/dub-link-shortener-1.png:1]: yeah, I agree. the renaming happened in a different PR, ill ask Steven to sort that out. This PR is only transferring to an organisation.

Comment (jfkisafk) [reply extensions/dub/metadata/dub-link-shortener-1.png:1]: cool @peduarte ! I fixed it in an adjacent PR: https://github.com/raycast/extensions/pull/13824

**Pull request merged**